src/libraries/Microsoft.PowerFx.Core/Types/DType.cs

```diff
@@ -1923,7 +1923,6 @@ public virtual bool Accepts(DType type, out KeyValuePair<string, DType> schemaDi
                         type.Kind == DKind.Blob ||
                         type.Kind == DKind.Unknown ||
                         type.Kind == DKind.Deferred ||
-                        type.Kind == DKind.Guid ||
```

Comment:
>  type.Kind == DKind.Guid || [](http://example.com/codeflow?start=22&length=28)

From PA customer POV, this would be a transparent change. Is that accurate?

Reply:
Correct. But we'll need to update this PR to prevent analysis breaking changes.